Snakefile-sorting-hat

```diff
@@ -252,9 +252,12 @@ rule produce_recalculations_call:
         mkdir -p {wildcards.accession}/logs
         exec &> {log}
         pushd {input.working_directory}
-	if ! [ -z {params.lsf_config} ]; then 
+        if ! [ -z {params.lsf_config} ]; then
+            if [ -f "lsf.yaml" ]; then
+                rm -f lsf.yaml
+            fi
             ln -s {params.lsf_config} lsf.yaml
```

Comment:
```suggestion
            if [ ! -f "lsf.yaml" ]; then
                ln -s {params.lsf_config} lsf.yaml
            fi
```
might be less expensive to do this instead of deleting and creating every time.